examples/multi/bundle/src/App.js

```diff
@@ -21,71 +21,58 @@
 
 import React, { Component } from "react";
 import "regenerator-runtime/runtime";
+import { SessionManager } from "../../../../dist/index";
 
-import fetcher, {
-  login,
-  getSession,
-  handleRedirect,
-  logout,
-  fetch
-} from "../../../../../dist/index";
-
-class Form extends Component {
+class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
       status: "loading",
-      loginIssuer: "https://localhost:8443",
-      fetchRoute: "https://jackson.localhost:8443/private",
+      loginIssuer: "https://dev.inrupt.net",
+      fetchRoute: "https://jackson.dev.inrupt.net/private",
```

Comment:
I know this is just 'example' code, but firstly I would use a really name in here (i.e. 'jackson'), I'd just use 'example-user' or something). Separately, I think we should consider having (later, not now) a dedicated ESS instance setup and running purely for serving examples like these - i.e. where a developer could literally run this example code as-is and it would 'just-work'.
So in fact maybe I'd change the IRI here to be "https://ldp.example-ess.inrupt.com/example-user/private" (and the 'loginIssuer' to be "https://identity.example-ess.inrupt.com/")... (and probably create an Inrupt Example vocab (alongside the current Inrupt Test Vocab (https://github.com/inrupt/lit-vocab/blob/master/inrupt-rdf/Common/CopyOfVocab/inrupt-test.ttl) for these example values)).

Reply:
Absolutely, I think that fits right into the hardening tickets we'd like to see between beta and GA